### ISSUE TITLE
NAIP Mosaic Dataset loses Time setting during upgrade to 10.4.1

### ISSUE THREAD

**Issue by kerryrobinson** [Esri/ops-server-config#1484]
I noticed after I upgraded my Ops Server to 10.4.1 that the NAIP image service wasn't appearing as time-enabled in a web map.   This setting was no longer set on the mosaic dataset.

I had to update this setting in the properties for the mosaic dataset (C:\OpsServer\Data\FoundationOverlays\OrthophotoBase\NAIP\NAIP.gdb\NAIP).

Once I did that, I didn't need to make any additional changes to the web maps containing NAIP; they were automatically time-enabled. 

I'm not sure if this has happened to any other mosaic datasets.

**Comment by elinz** [comment]
@kerryrobinson I made the following changes to the NAIP mosaic dataset:
Use Time: Yes; Start Time Field: Year; End Time Field: Year; Time Value Format: YYYY; Time Interval Units: Years.

There is a Time Interval Value field, but not sure if it's supposed to contain a value or what the value should be. Whether I set to value as 1 or leave empty the time slider shows each interval as 2 years, i.e.:
December 31, 2004 to December 31, 2006
December 31, 2006 to December 31, 2008
December 31, 2008 to December 31, 2010
December 31, 2010 to December 31, 2012
December 31, 2012 to December 31, 2014

Not sure if this is correct. Let me know and I can make the necessary changes and then close this issue.
